### ISSUE TITLE
fatal error: 'webp/decode.h' file not found

### ISSUE THREAD

**Issue by firminochangani** [kolesa-team/go-webp#19]
Hi there,

I've just installed `go-webp` through the following steps on a MacOS M1:

1 - `brew install webp`
2 - `go get -u github.com/kolesa-team/go-webp`

When I try to run the demo, I get the following error:

```
.../github.com/kolesa-team/go-webp@v1.0.1/decoder/decoder.go:27:10: fatal error: 'webp/decode.h' file not found
#include <webp/decode.h>
         ^~~~~~~~~~~~~~~
1 error generated.
# github.com/kolesa-team/go-webp/encoder
.../go/pkg/mod/github.com/kolesa-team/go-webp@v1.0.1/encoder/encoder.go:27:10: fatal error: 'webp/encode.h' file not found
#include <webp/encode.h>
         ^~~~~~~~~~~~~~~
1 error generated.

```

Thanks in advance.

**Comment by linksquidmx** [comment]
You must download the libwebp library for your OS from its official site, unzip, and copy the "webp" folder that is inside "Include" and paste it into the directory "GOPATH\pkg\mod\github.com\kolesa-team\go-webp@v1.0.1\decoder" and "GOPATH\pkg\mod\github.com\kolesa-team\go-webp@v1.0.1\encoder". If you have additional problems, I recommend migrating to another package: **chai2010/webp**. Or better yet, to one that uses the precompiled libwebp library and no longer requires using cgo to compile C code: **nickalie/go-webpbin**

**Comment by Lehp** [comment]
This problem is not OS specific. Bugs out on linux and macos atleast. Also this lib once worked perfectly on macos idk

**Comment by fabiomarini** [comment]
This problem is resolved by #22 which has been integrated into version 1.0.3
